docs/sources/operators-guide/configuring/configuring-out-of-order-samples-ingestion.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Configuring out-of-order samples ingestion"
+menuTitle: "Configuring out-of-order samples ingestion"
+description: "Learn how to configure Grafana Mimir to handle out-of-order samples ingestion."
+weight: 120 
+---
+
+# Configuring out-of-order samples ingestion
+
+By default Grafana Mimir will only accept new samples that are newer than the last ingested samples for a series. Those that are older than the last sample that was ingested are called out-of-order samples.
+
+If you happen to have out-of-order samples due to the nature of your architecture or the system that is being observed, then you can configure Grafana Mimir to set an allowance threshold for how old samples can be ingested.
+
+### Configuration
```

Comment:
Main section is already "Configuring out-of-order ...", I would drop this section title.